example/src/WebViewExample.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Section, { Container } from "./Section";
+import { WebView } from "@draftbit/ui";
+import { Dimensions, Platform, StyleSheet } from "react-native";
+
+const WebViewExample = () => (
+  <Container>
+    <Section title="With URI">
+      <WebView
+        style={style.webView}
+        source={{
+          uri: "https://docs.expo.io/",
+        }}
+      />
+    </Section>
+    <Section title="With custom HTML">
```

Comment:
custom html won't work for our platform, you can remove it

Reply:
Do you mean the implementation or the example? (or both)

Reply:
I suppose for the sake of example we can leave it!